docs/tutorials/mac_experimental.md

```diff
@@ -0,0 +1,87 @@
+# Using podman-machine on MacOS (x86_64 and Apple silicon)
+
+## SetUp
+
+You must obtain a compressed tarball that contains the following:
+* a qcow image
+* an app.entitlements file
```

Comment:
```suggestion
* an entitlements.plist file
```

Reply:
actually i think this can be deleted now that we use the qemu one. agree?

Reply:
I didn't find a qemu one in the repo. so I made my own